lib/KeyboardAwareHOC.js

```diff
@@ -273,12 +273,24 @@ function KeyboardAwareHOC(
 
     scrollToPosition = (x: number, y: number, animated: boolean = true) => {
       const responder = this.getScrollResponder()
-      responder && responder.scrollResponderScrollTo({ x, y, animated })
+      if (responder.scrollTo) {
```

Comment:
I tested this on 0.65 and 0.63. If it exists then it should perform correctly. I can swap it if you would prefer though. I don't think it should make much of a difference here. 

Also I'm wondering how far back the community should be supporting older versions of RN.
To this point, this library has [documentation](https://github.com/APSL/react-native-keyboard-aware-scroll-view#supported-versions) on which version of the lib support older versions of RN so we don't need to cater to ones so far back and it's shown as not uncommon to split the version that this supports.

I would prefer to keep this approach since it's aimed towards where RN is headed in support and just do some research how far back this stops working and add to the documentation to inform users about the break in experience.

Reply:
@mingtsay updated to swap them.